### PR TITLE
[Excel] (Custom functions) Fix image and delete Insider mentions

### PR DESCRIPTION
**Pull request by mscharlock**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[5d6757e](https://github.com/OfficeDev/office-js-docs-pr/pull/1015/commits/5d6757e392be5a3b8a1d3e828da704f3b3fd919c)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-fixsphere/docs/excel/custom-functions-overview.md) | :warning:Warning | | [Details](#user-content-e1faa9e0-e3940a7f)
[docs/excel/custom-functions-requirements.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-fixsphere/docs/excel/custom-functions-requirements.md) | :white_check_mark:Succeeded | |
[docs/excel/xll-compatibility-requirements.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-fixsphere/docs/excel/xll-compatibility-requirements.md) | :white_check_mark:Succeeded | |
[docs/images/SphereVolumeNew.gif](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-fixsphere/docs/images/SphereVolumeNew.gif) | :white_check_mark:Succeeded | |

<a id="e1faa9e0-e3940a7f"></a>
### [docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-fixsphere/docs/excel/custom-functions-overview.md)
 - **Line 0**: **[Warning]** `````Invalid file link:(~/images/SphereVolumNew.gif).`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C16%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201905160014593399-1015%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=wIrIXmZwqwVTyhMn2mT%2FxB1jwHSGpZ%2FZ4gHzUqVkThQ%3D&st=2019-05-16T00%3A11%3A36Z&se=2019-06-16T00%3A16%3A36Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/excel/custom-functions-overview.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Custom functions enable developers to add new functions to Excel by defining tho
 
 The following animated image shows your workbook calling a function you've created with JavaScript or Typescript. In this example, the custom function `=MYFUNCTION.SPHEREVOLUME` calculates the volume of a sphere.
 
-<img alt="animated image showing an end user inserting the MYFUNCTION.SPHEREVOLUME custom function into a cell of an Excel worksheet" src="../images/SphereVolume.gif" />
+<img alt="animated image showing an end user inserting the MYFUNCTION.SPHEREVOLUME custom function into a cell of an Excel worksheet" src="../images/SphereVolumNew.gif" />
```
Comment (ElizabethSamuel-MSFT) [inline docs/excel/custom-functions-overview.md]:
```suggestion
<img alt="animated image showing an end user inserting the MYFUNCTION.SPHEREVOLUME custom function into a cell of an Excel worksheet" src="../images/SphereVolumeNew.gif" />
```

Comment (ElizabethSamuel-MSFT) [reply docs/excel/custom-functions-overview.md]: Also, do you need to delete the old image file?

**Comment by o365devx** [comment]
Docs Build status updates of commit _[c8e38d8](https://github.com/OfficeDev/office-js-docs-pr/pull/1015/commits/c8e38d8bb21e87f1f51ff1a95ec276e26a75590b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-fixsphere/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-requirements.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-fixsphere/docs/excel/custom-functions-requirements.md) | :white_check_mark:Succeeded | |
[docs/excel/xll-compatibility-requirements.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-fixsphere/docs/excel/xll-compatibility-requirements.md) | :white_check_mark:Succeeded | |
[docs/images/SphereVolumeNew.gif](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-fixsphere/docs/images/SphereVolumeNew.gif) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C16%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201905161848294779-1015%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=MB5v%2B2RX18sfjljqAIeSdE8EmynYCf9Ni1%2BKa%2FesIG8%3D&st=2019-05-16T18%3A45%3A07Z&se=2019-06-16T18%3A50%3A07Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/excel/custom-functions-overview.md (hunk 1 above)
Comment (mscharlock) [reply docs/excel/custom-functions-overview.md]: Will do!

**Comment by o365devx** [comment]
Docs Build status updates of commit _[047fb03](https://github.com/OfficeDev/office-js-docs-pr/pull/1015/commits/047fb031a4aefd575275338236cf909b3a9943ae)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-fixsphere/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-requirements.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-fixsphere/docs/excel/custom-functions-requirements.md) | :white_check_mark:Succeeded | |
[docs/excel/xll-compatibility-requirements.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-fixsphere/docs/excel/xll-compatibility-requirements.md) | :white_check_mark:Succeeded | |
[docs/images/SphereVolumeNew.gif](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-fixsphere/docs/images/SphereVolumeNew.gif) | :white_check_mark:Succeeded | |
[docs/images/UpdatedYoOfficePrompt.PNG](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-fixsphere/docs/images/UpdatedYoOfficePrompt.PNG) | :white_check_mark:Succeeded | |
[docs/quickstarts/excel-custom-functions-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-fixsphere/docs/quickstarts/excel-custom-functions-quickstart.md) | :white_check_mark:Succeeded | |
[docs/tutorials/excel-tutorial-create-custom-functions.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-fixsphere/docs/tutorials/excel-tutorial-create-custom-functions.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C16%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201905161906533443-1015%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=jQHo41sGRmBWV9IofB1Yo%2BzsLTmJ5mFucRBknDXweCk%3D&st=2019-05-16T19%3A03%3A24Z&se=2019-06-16T19%3A08%3A24Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/excel/xll-compatibility-requirements.md [hunk 2]
```diff
@@ -16,9 +16,7 @@ COM add-in and XLL UDF compatibility is currently available on the following pla
 To use COM add-in and XLL UDF compatibility within Excel Online, login by using either your Office 365 subscription or a [Microsoft account](https://account.microsoft.com/account).
 
 ## Excel on Windows and Excel for Mac
-To use COM add-in and XLL UDF compatibility within Excel on Windows or Excel for Mac, you must have an Office 365 subscription, join the [Office Insider](https://products.office.com/office-insider) program (**Insider** level -- formerly called "Insider Fast"), and use a sufficiently recent build of Excel (as specified previously).
-
-If you are using a version of Office on your desktop which you downloaded from the Windows Store, you must be part of the [Windows Insider](https://insider.windows.com/) program at the **Insider** level (formerly called "Insider Fast"), running the April 2018 Update version or later to use COM add-in and XLL UDF compatibility. This is a new change as of January 2019.
+To use COM add-in and XLL UDF compatibility within Excel on Windows or Excel for Mac, you must have an Office 365 subscription and use a sufficiently recent build of Excel (as specified previously).
```
Comment (davidchesnut) [inline docs/excel/xll-compatibility-requirements.md]: This still requires insider builds.

**Comment by davidchesnut** [comment]
Also @mscharlock can you please remove the insider requirement from the quick start and tutorial as part of this PR? Thanks!

**Comment by mscharlock** [comment]
Yep, will fix. Tutorial and QS are in this pr already :)

**Comment by o365devx** [comment]
Docs Build status updates of commit _[e9aeebe](https://github.com/OfficeDev/office-js-docs-pr/pull/1015/commits/e9aeebe5465f72a2972724cd921af59496f7e8ce)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-bfed2566)

<a id="bfed2566"></a>
###
 - **[Error]** 
`````
Cannot sync git repo to specified commit because branch michelle-fixsphere has been deleted or has been force pushed: fatal: Couldn't find remote ref michelle-fixsphere

`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C22%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201905221902301182-1015%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Vql%2BdG%2FLpLgzeQM3MdQ3uede%2BrjkH66CJ4rGrExchOQ%3D&st=2019-05-22T18%3A57%3A46Z&se=2019-06-22T19%3A02%3A46Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**